cal/schema.py

```diff
@@ -43,6 +42,7 @@ class User(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String, unique=True, nullable=False)
```

Comment:
I think keeping a separate Id is a good idea, since it makes for cleaner queries (and if we ever make a public API, keeping a simple integer ID would be clean).